### PR TITLE
 D2K - Add Ordos Mission 4

### PR DESCRIPTION
**Pull request by MustaphaTR**

### CODE REVIEW

File: mods/d2k/maps/ordos-04/ordos04.lua [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+end
+
+InitObjectives = function()
+	Trigger.OnObjectiveAdded(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+
+	KillOrdosH = harkonnen.AddPrimaryObjective("Kill all Ordos units.")
+	KillOrdosS = smuggler.AddSecondaryObjective("Kill all Ordos units.")
+	DefendOutpost = smuggler.AddPrimaryObjective("Don't let the outpost to be captured or dostroyed.")
```
Comment (ltem) [inline mods/d2k/maps/ordos-04/ordos04.lua]: Small typo dostroyed -> destroyed

**Comment by MustaphaTR** [comment]
Updated.

**Review by ltem — APPROVED** [review]

**Comment by ltem** [comment]
:+1: 
- Code is fine
- All the objectives work

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d067edd38d9db0c63dd56f7609c49eb18538117f...d70db4b86ada77eae4b50d886aeeccd69945c368)

**Pull request merged**